utils/sortSearch.tsx

```diff
@@ -0,0 +1,6 @@
+export const sortSearch = (values: string[], keyword: string) =>
```

Comment:
This should probably have a more generic name to not lock it down to one functionality.

Reply:
Do you have any suggestions? Since it sorts an array of values based on a search term I thought sortSearch was pretty generic.

Reply:
Good point. I didn't see the keyword 🤠